tests/sentry/api/endpoints/test_organization_root_cause_analysis.py

```diff
@@ -488,11 +488,13 @@ def test_geo_code(self):
                 "duration_before": 10.0,
                 "duration_after": 100.0,
                 "duration_delta": 90.0,
+                "score": 0.08333333333333334,
```

Comment:
I'm not sure if I should expose `score` here or not. I don't expect to use it in the frontend but I think it's useful for cases of debugging?

Reply:
I think it'd be useful for debugging for sure